HelloDotNet/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Repository Maintainers
+* @launchdarkly/team-sdk-net
```

Comment:
The labels need updated for this to be valid.

Reply:
Not sure is this just labels or I need to have this repo managed-by-terraform to get the full result. This I will do in a separate PR.